depthmapX/glview.cpp

```diff
@@ -36,29 +36,45 @@ GLView::GLView(QWidget *parent, QGraphDoc* doc, const QRgb &backgroundColour, co
     m_core = QCoreApplication::arguments().contains(QStringLiteral("--coreprofile"));
     m_pDoc = doc;
 
+    if(m_multiSampleSamples) {
+        QSurfaceFormat format;
+        format.setSamples(m_multiSampleSamples);    // Set the number of samples used for multisampling
+        setFormat(format);
+    }
 
     loadDrawingGLObjects();
 
     loadAxes();
 
-    if(m_pDoc->m_meta_graph->getViewClass() & m_pDoc->m_meta_graph->VIEWAXIAL) {
+    if(m_pDoc->m_meta_graph->getViewClass() & MetaGraph::VIEWAXIAL) {
         m_visibleShapeGraph.loadGLObjects(m_pDoc->m_meta_graph->getDisplayedShapeGraph());
     }
     m_visiblePointMap.setGridColour(colorMerge(foregroundColour, backgroundColour));
-    if(m_pDoc->m_meta_graph->getViewClass() & m_pDoc->m_meta_graph->VIEWVGA) {
+    if(m_pDoc->m_meta_graph->getViewClass() & MetaGraph::VIEWVGA) {
         m_visiblePointMap.loadGLObjects(m_pDoc->m_meta_graph->getDisplayedPointMap());
     }
 
-    if(m_pDoc->m_meta_graph->getViewClass() & m_pDoc->m_meta_graph->VIEWDATA) {
+    if(m_pDoc->m_meta_graph->getViewClass() & MetaGraph::VIEWDATA) {
         m_visibleDataMap.loadGLObjects(m_pDoc->m_meta_graph->getDisplayedDataMap());
     }
+    PafColor pafForegroundColour;

```

Comment:
Is it technically necessary to go via PafColour here? Or just to be in line with dX idioms? Maybe we should break PafColor out of vertex.h/cpp and make it a first class member of the either sala or genlib?